dev/com.ibm.ws.security.saml.websso.2.0/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -134,6 +134,9 @@ nameIDFormat.CUSTOMIZE=Customized Name ID Format.
 customizeNameIDFormat=Customized XML name space of name id format
 customizeNameIDFormat.desc=Specifies the customized URI reference corresponding to a name identifier format that is not defined in the SAML core specification.
 
+disableIRCookie=Disable Initial Request cookie creation
+disableIRCookie.desc=Do not create the initial request cookie if there are too many authentications requests from Service Provider to IdP as a result of the application SSO setup.
```

Comment:
```suggestion
disableIRCookie.desc= When too many authentications requests are received from Service Provider to IdP due to the application SSO setup, set this attribute to true to prevent creation of the initial request cookie. The default is false.
```

Reply:
not sure if disableIRCookie is an attribute, property or element- please adjust suggestion as necessary

Reply:
it is an attribute. I adjusted the wording